number_parser/parser.py

```diff
@@ -114,15 +116,17 @@ def tokenize(input_string):
     tokens = re.split(r'(\W)', input_string)
     return tokens
 
-def parse_number(input_string):
+def parse_number(input_string, lang='en'):
```

Comment:
Adding this comment here since I am not able to add at required line
For [line](https://github.com/arnavkapoor/number-parser/pull/12/files?file-filters%5B%5D=.py&file-filters%5B%5D=No+extension#diff-634e39388755c9ef23f50299dc52faf5R112)
Should we have constants at start of the file after imports so they are easy to track and manage?